### PR TITLE
[jackett] Migrate to media-common

### PR DESCRIPTION
**Pull request by bjw-s**
Contributes to #37 

#### Special notes for your reviewer:
n/a

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/jackett/README.md [hunk 1]
```diff
@@ -28,82 +28,35 @@ helm delete my-release --purge
...
+Read through the media-common [values.yaml](https://github.com/k8s-at-home/charts/blob/master/charts/media-common/values.yaml)
+file. It has several commented out suggested values.
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
-
 ```console
-helm install --name my-release \
-  --set timezone="America/New York" \
+helm install jackett \
+  --set radarr.env.TZ="America/New York" \
```
Comment (billimek) [inline charts/jackett/README.md:37]: This should be `--set jackett.env.TZ="America/New York" \`

File: charts/jackett/ci/ct-values.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+jackett:
+  image:
+    organization: linuxserver
+    repository: jackett
+    tag: latest
```
Comment (billimek) [inline charts/jackett/ci/ct-values.yaml:5]: Should we use the same version as is in the `values.yaml` (`v0.16.1045-ls14`) here?  Floating to latest may have unintended consequences when testing an unrelated change to the chart and latest happens to have some breaking change.

File: charts/jackett/values.yaml [hunk 3]
```diff
@@ -1,149 +1,47 @@
...
-
-    ## If subPath is set mount a sub folder of a volume instead of the root of the volume.
-    ## This is especially handy for volume plugins that don't natively support sub mounting (like glusterfs).
-    ##
-    subPath: ""
-    ## Do not delete the pvc upon helm uninstall
-    skipuninstall: false
-  torrentblackhole:
+  # Configure the OpenVPN add-on
+  openvpn:
```
Comment (billimek) [inline charts/jackett/values.yaml:19]: Is it necessary to have the disabled `openvpn` values stub here?  We don't include stubs for the other things from media-common inverse-umbrella chart, unless I'm misunderstanding how the relationship between this chart and the media-common works?

**Review by billimek — COMMENTED** [review]
Just a few comments/suggestions

**Comment by bjw-s** [comment]
Good points! Will fix them when I get back from the weekend
